packages/@react-spectrum/list/src/InsertionIndicator.tsx

```diff
@@ -20,6 +20,10 @@ export default function InsertionIndicator(props: InsertionIndicatorProps) {
 
   let isDropTarget = dropState.isDropTarget(target);
 
+  if (!isDropTarget && dropIndicatorProps['aria-hidden']) {
+    return null;
+  }
```

Comment:
Remind me, what was the reason for not rendering the drop indicators until a drag session starts?

Reply:
I think it makes sense, no reason to have a drop button in the DOM if nothing is being dropped. Some background: https://github.com/adobe/react-spectrum/blob/a0a88da217db1cff67be1bdc94490b3a7002b6f3/packages/%40react-aria/dnd/stories/DroppableGrid.tsx#L329-L331

Reply:
Also https://github.com/adobe/react-spectrum/pull/3055#discussion_r870460265

Reply:
I see, thanks. Yeah that makes sense to me, just wanted to make sure there wasn't some weird bug that was also being handled by this change